test/test_mps.py

```diff
@@ -10171,6 +10172,9 @@ def get_samples():
                 self.assertEqual(cpu_out, mps_out, atol=atol, rtol=rtol)
 
             except Exception as e:
+                if any(s in str(e).lower() for s in ["float16", "div truc rounding"]):
```

Comment:
This will be cleaned out when we move to XFAILLIST. It will be next PR, so I didn't want to spend time on moving it to xfaillist.

Reply:
Sure, as part of the XFAILLIST it will be removed.